credsweeper/filters/value_grafana_check.py

```diff
@@ -0,0 +1,38 @@
+import base64
+import contextlib
+import json
+
+from credsweeper.credentials import LineData
+from credsweeper.filters import Filter
+
+
+class ValueGrafanaCheck(Filter):
+    """Grafana Provisioned API Key and Access Policy Token"""
+
+    def run(self, line_data: LineData) -> bool:
+        """Run filter checks on received token which might be structured.
+
+        Args:
+            line_data: credential candidate data
+
+        Return:
+            True, when need to filter candidate and False if left
+
+        """
+        if not line_data.value:
+            return True
+        with contextlib.suppress(Exception):
+            if line_data.value.startswith("glc_"):
+                # Grafana Access Policy Token
+                decoded = base64.b64decode(line_data.value[4:])
+                keys = ["o", "n", "k", "m"]
```

Comment:
What about check like below? For more accuracy.

```suggestion
                keys = ['"o":', '"n":', '"k":', '"m":']
```

Reply:
The payload is parsed anyway. Yeys searching in dictionary is quickly.  There might be broken json. 
Additionally json allows constructions ``"k" :`` - with a space.

Reply:
Then we will skip strong verification with parsing Json structure. It might takes FP cases(very low probability, but it is)

Reply:
NVM I misunderstood about this.... :tired_face: